src/engine/filters/enginefiltermoogladder4.h

```diff
@@ -93,11 +93,12 @@ class EngineFilterMoogLadderBase : public EngineObjectConstIn {
         // Resonance correction for self oscillation ~4
         m_kacrNew = resonance * (-3.9364f * (kfc * kfc) + 1.8409f * kfc + 0.9968f);
 
+        // See https://github.com/mixxxdj/mixxx/pull/11177 for the Jupyter
+        // notebook used to derive this
         if (MODE == MoogMode::HighPassOversampling || MODE == MoogMode::HighPass) {
-            m_postGainNew = 1;
+            m_postGainNew = 0.8478175496499384f + (0.09022626934231257f * resonance);
```

Comment:
Strange, these values makes the gain dependency worse. 
Why do we see different results?

Reply:
I reran the numbers for a more aggressive cutoff frequency (0.001 Hz instead of 1 Hz) and that that got me the following polynomial:

![image](https://user-images.githubusercontent.com/748520/210459354-703aeae8-04e5-4ecc-9f0c-f910fb8a9ee6.png)

Which is the 1.0 value that the filter used before this PR. Going back to this (or the above polynomial) does sound correct to me. When did you notice a problem with this? Note that I'm talking about the _perceived_ loudness. You will obviously see your VU meter increase a bit when you start increasing the high-pass filter's frequency. This is to be expected because of the resonant bump around the cutoff frequency. At low cutoff values the low frequencies are amplified while the other frequencies in the filter's pass band stay at equal levels. So a 1.0 post gain compensation value for the HPF is correct here.

Reply:
I have recorded a sweep sound generated by Audacity. One time without the filter and one time with the filter active. Than I have compared the peak samples in the pass-band. I will repeat this tonight.

Reply:
Try playing regular music. I optimized these curves so that when the filter gets engaged from its neutral position (the highest cutoff frequency for the low-pass filter, and the lowest cutoff frequency for the high-pass filter) there is no perceived gain different for most music.